algorithms/ar-bsrh/java/SieveOfEratosthenes.java

```diff
@@ -0,0 +1,37 @@
+import java.util.*;
+
+public class SieveOfEratosthenes {
```

Comment:
Codacy found an issue: [The utility class name 'SieveOfEratosthenes' doesn't match '[A-Z][a-zA-Z0-9]+(Utils?|Helper|Constants)'](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6278826)